docker/entrypoint.bash

```diff
@@ -36,9 +36,8 @@ ls -lah "${DY_SIDECAR_PATH_OUTPUTS}"
 echo "setting correct user id/group id..."
 HOST_USERID=$(stat -c %u "${DY_SIDECAR_PATH_INPUTS}")
 HOST_GROUPID=$(stat -c %g "${DY_SIDECAR_PATH_INPUTS}")
-# since below command hangs we disbale it and set the group to none to create it
-#CONT_GROUPNAME=$(getent group "${HOST_GROUPID}" | cut -d: -f1)
-CONT_GROUPNAME=""
+CONT_GROUPNAME=$(getent group | grep "${HOST_GROUPID}" | cut -d: -f1 || echo "")
```

Comment:
is this running in an ubuntu/debian based image? if yes can we have verbose options?

Reply:
Yes and Yes.